components/collective-page/index.js

```diff
@@ -273,7 +273,7 @@ class CollectivePage extends Component {
           showSelectedCategoryOnMobile
         />
 
-        <div ref={this.sectionsContainerRef}>
+        <div ref={this.sectionsContainerRef} className="sectionsContainer">
```

Comment:
`className` is not the proper way to target things with `StyledComponents` we still have some occurrences in the code that were used for retro-compatibility, but the proper thing to do in these case is to define a dedicated component

```jsx
const SectionsContainer = styled.div` ... `
```

Reply:
Thanks. I'll keep this in mind.